packages/flame/lib/camera.dart

```diff
@@ -0,0 +1,23 @@
+/// Classes and components in this sub-module are considered experimental,
+/// that is, their API may still be incomplete and subject to change at a more
+/// rapid pace than the rest of the Flame code.
+///
+/// However, do not feel discouraged to use this functionality: on the contrary,
+/// consider this as a way to help the Flame community by beta-testing new
+/// components.
+///
+/// After the components lived here for some time, and when we gain more
+/// confidence in their robustness, they will be moved out into the main Flame
+/// library.
```

Comment:
This comment should probably be removed now.

Reply:
Good catch!